apps/passport-server/src/database/queries/pretixItemInfo.ts

```diff
@@ -29,6 +29,7 @@ export async function insertPretixItemsInfo(
     `\
       insert into devconnect_pretix_items_info (item_id, devconnect_pretix_events_info_id, item_name)
       values ($1, $2, $3)
+      on conflict (item_id, devconnect_pretix_events_info_id) do update set item_name = $3
```

Comment:
do you think this needs to be tested? cc @robknight

Reply:
The application layer code already tries to avoid conflicts, so it would be very hard to write a test that uses the sync code _and_ which causes a conflict. `syncEventItemInfos` does a `select` query to decide whether to call `insertPretixItemsInfo` or `updatePretixItemsInfo`. Perhaps we could refactor that since `insertPretixItemsInfo` now does upserts, but that seems way out of scope for this PR. So long as the existing sync tests still pass, I don't see a need to add a further test here.